### PR TITLE
Catalan and Spanish translations

### PR DESCRIPTION
**Pull request by fitojb**
This pull request completes the existing Spanish translation and provides a brand-new Catalan translation.

### CODE REVIEW

**Comment by fitojb** [comment]
@Philip-Scott Ping?

File: po/ca.po [hunk 1]
```diff
@@ -0,0 +1,409 @@
...
+msgstr "Spice-Up"
+
+#: /home/felipe/Code/com.github.philip-scott.spice-up/po/../src/Application.vala:30
+msgid "About Spice-Up"
+msgstr "Quant a l’Spice-Up"
+
+#: /home/felipe/Code/com.github.philip-scott.spice-up/po/../src/Services/FileManager.vala:34
+#: /home/felipe/Code/com.github.philip-scott.spice-up/po/../src/Services/GamepadSlideController.vala:166
+msgid "Cancel"
+msgstr "Cancel·la"
```
Comment (Philip-Scott) [inline po/ca.po]: There's a dot in the middle of the words, is this ok?

File: po/ca.po [hunk 2]
```diff
@@ -0,0 +1,409 @@
...
+msgstr "Localitza el fitxer…"
+
+#: /home/felipe/Code/com.github.philip-scott.spice-up/po/../src/Widgets/Library/LibraryItem.vala:123
+msgid "Could not rename: File already exists…"
+msgstr "No s’ha pogut canviar el nom: el fitxer ja existeix…"
+
+#: /home/felipe/Code/com.github.philip-scott.spice-up/po/../src/Widgets/PresenterView.vala:228
+#, c-format
+msgid "%d of %d"
+msgstr "%d/%d"
```
Comment (Philip-Scott) [inline po/ca.po]: Can you add a space between the %d and the `/` :) Just so the numbers aren't stuck together

**Comment by Philip-Scott** [comment]
@fitojb pong! :sweat_smile: Sorry about taking so long to review. between school and work i don't get very much free time :/ But thank you for your contribution! I left a comment and a question

File: po/ca.po (hunk 1 above)
Comment (fitojb) [reply po/ca.po]: Yep. ⟨l·l⟩ is a digraph in Catalan. See https://en.wikipedia.org/wiki/Interpunct#Catalan

File: po/ca.po (hunk 2 above)
Comment (fitojb) [reply po/ca.po]: Sure; will add a thin space (U+202F).

Comment (Philip-Scott) [reply po/ca.po]: Why not just a normal space? :)

File: po/ca.po (hunk 1 above)
Comment (Philip-Scott) [reply po/ca.po]: Sounds goo! Thank you for confirming it's correct!

**Comment by fitojb** [comment]
Branch updated :)

**Comment by Philip-Scott** [comment]
Thanks again for your work with the translations! Take care man!

**Pull request merged**